docs/source/discord/resources/channel/methods/list_thread_members.rst

```diff
@@ -6,14 +6,21 @@ List Thread Members
 Example
 -------
 
-.. code:: c
+.. code:: c   
+   
+   void done_list_thread_members(struct discord *client, void *data, const struct discord_thread_members *ret)
+   {
+     if (ret)
+       for (int i = 0; ret[i]; ++i)
+         printf("%"PRIu64"\n", ret[i]->user_id);
```

Comment:
Please check the [guide](https://gist.github.com/lcsmuller/a5f2b205c3871888656b86825db90187#example-3) for the new method of iterating our built-in array types.